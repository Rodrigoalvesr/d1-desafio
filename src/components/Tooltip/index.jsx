import MuiTooltip from '@material-ui/core/Tooltip';
import { withStyles } from '@material-ui/core/styles';

const Tooltip = withStyles((theme) => ({
  tooltip: {
    color: '#9D9D9D',
    fontSize: 12,
    fontWeight: 'bold',
    padding: theme.spacing(1),
    boxShadow: theme.shadows[1],
    backgroundColor: theme.palette.common.white,
  },
  arrow: {
    color: theme.palette.common.white,
  },
}))(MuiTooltip);

export default Tooltip;
