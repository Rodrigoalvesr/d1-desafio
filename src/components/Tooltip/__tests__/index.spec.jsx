import React from 'react';
import renderer from 'react-test-renderer';
import Tooltip from '../index';

describe('Tooltip', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(
        <Tooltip title="teste">
          <div>teste</div>
        </Tooltip>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
