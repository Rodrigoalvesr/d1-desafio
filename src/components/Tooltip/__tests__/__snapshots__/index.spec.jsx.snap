// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip Renders correctly 1`] = `
<div
  aria-describedby={null}
  className=""
  onBlur={[Function]}
  onFocus={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  title="teste"
>
  teste
</div>
`;
